### PR TITLE
cherry-pick release script related commits to `branch-2.2.0`

### PR DESCRIPTION
**Pull request by j143**
To be applied cleanly as rebase!

cc @phaniarnab

### CODE REVIEW

**Comment by j143** [comment]
1. change the connection to personal fork 0f6c7f5
2. [maven-release-plugin] prepare release 2.2.1-rc1 0a6b7fa
3. [maven-release-plugin] prepare for next development iteration 298eea6

above verifies that the scripts run fine. of course, these three commit are **not for merging**.

**Comment by phaniarnab** [comment]
I see the changes are in the files to do the post-voting release.
Is the RC generation from a non-master branch working fine?

File: pom.xml [hunk 1]
```diff
@@ -68,7 +68,7 @@
 	</repositories>
 
 	<scm>
-		<developerConnection>scm:git:https://github.com/apache/systemds.git</developerConnection>
+		<developerConnection>scm:git:https://github.com/j143/systemds.git</developerConnection>
```
Comment (Baunsgaard) [inline pom.xml:71]: should this be changed?

**Review by Baunsgaard — COMMENTED** [review]
Some of the changes i think we should do in general:

- dev/release/pypi_upload.sh 
- dev/release/svn-staging-to-release.sh

the rest only on branch.

LGTM

**Comment by j143** [comment]
Thank you for the review. This PR is to check whether the release-scripts function as intended.

Arnab, requested not to merge any nice-to-have improvements. So, closing this one. :)

**Pull request closed without merging**